client/stylesheets/_responsive.scss

```diff
@@ -359,14 +351,32 @@
 }
 
 @media (max-width: 600px) {
-    
+    .profileCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardLi` should be written in lowercase with hyphens